derive_builder/src/lib.rs

```diff
@@ -367,6 +367,28 @@
 //! }
 //! ```
 //!
+//! ## Additional Trait Derivations
+//!
+//! You can derive additional traits on the builder, including traits defined by other crates:
+//!
+//! ```rust
+//! # #[macro_use]
+//! # extern crate derive_builder;
+//! #
+//! #[derive(Builder, Clone)]
+//! #[builder(derive(Debug, PartialEq, Eq))]
+//! pub struct Lorem {
+//!     foo: u8,
+//!     bar: String,
+//! }
+//! 
+//! fn main() {
+//!    assert_eq!(LoremBuilder::default(), LoremBuilder::default());
+//! }
+//! ```
+//!
+//! Attributes declared for those crates are _not_ forwarded to the fields on the builder.
```

Comment:
What do you mean by _crates_ here? Do you mean struct instead?

Please add another note, that we derive `Default` and `Clone` on the builder unconditionally. Well, unless we change that ^^_Alternatively_ this would be our chance to say `Default` and `Clone` are only derived automatically if the user didn't specify `#[builder(derive)]`. This would allow to opt-out of these derivations - but I feel like they don't do any harm. I lean towards unconditionally deriving them. What do you think?